File: Source/Fantasy.Jobs/Scheduling/CustomAction.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Runtime.Serialization;
using Fantasy.XSerialization;

namespace Fantasy.Jobs.Scheduling
{
    [DataContract]
    [XSerializable("customAction", NamespaceUri = Consts.ScheduleNamespaceURI)]
    public class CustomAction : Action
    {
        [DataMember]
        [XAttribute("customType")]
        public string CustomType { get; set; }

        public override ActionType Type
        {
            get { return ActionType.Custom; }
        }
    }
}

File: branches/Stheno/AutoConverted/Fantasy.Jobs/Scheduling/CustomAction.java
package Fantasy.Jobs.Scheduling;

import Fantasy.XSerialization.*;

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
//[DataContract, XSerializable("customAction", NamespaceUri = Consts.ScheduleNamespaceURI)]
public class CustomAction extends Action
{
//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
	//[DataMember, XAttribute("customType")]
	private String privateCustomType;
	public final String getCustomType()
	{
		return privateCustomType;
	}
	public final void setCustomType(String value)
	{
		privateCustomType = value;
	}

	@Override
	public ActionType getType()
	{
		return ActionType.Custom;
	}
}